test/integration/framework.go

```diff
@@ -76,7 +76,7 @@ func withConfigGetFreshApiserverAndClient(
 
 	var etcdOptions *server.EtcdOptions
 	etcdOptions = server.NewEtcdOptions()
-	etcdOptions.StorageConfig.ServerList = serverConfig.etcdServerList
+	etcdOptions.StorageConfig.Transport.ServerList = serverConfig.etcdServerList
```

Comment:
Same thing, it just got moved into a sub-struct on the StorageConfig

Reply:
I find that a bizarre name. it's not a transport or anything like a transport.

que sera